contrib/custom-image-kubelet/README.md

```diff
@@ -24,7 +24,7 @@ The remaining RKE2 components include:
 Requirements
 ----
 
-1. RKE2 v1.18.13+rke2r1 or newer (with image/kubelet override support)
+1. RKE2 v1.18.15+rke2r1 or newer (with image/kubelet override support)
```

Comment:
This one doesn't need to be changed, since we're just telling users what version this feature was added.

Reply:
Updated.